config/api.yml

```diff
@@ -621,6 +621,12 @@
         :identifier: storage_tag
       - :name: unassign
         :identifier: storage_tag
+  :ems_events:
```

Comment:
We are looking to use the provider to expose events from the endpoint so that they can be used by Automate and Control.   Is there an alternate way to expose events coming from the end point? This is probably worth a discussion.